fairlearn/metrics/_group_metric_set.py

```diff
@@ -1,6 +1,7 @@
 # Copyright (c) Microsoft Corporation. All rights reserved.
 # Licensed under the MIT License.
 
+from bunch import Bunch
```

Comment:
You could alternatively just use sklearn's bunch (`sklearn.utils`). The bunch package is not really maintained anyway.

Reply:
Can you point me at the docs? The `sklearn` FAQ isn't keen on that:
https://scikit-learn.org/stable/faq.html?highlight=sklearn%20utils%20bunch#how-can-i-create-a-bunch-object
and I can't find 'bunch' on:
https://scikit-learn.org/stable/modules/classes.html#module-sklearn.utils
Thanks for the note about the maintenance of the 'bunch' package - I hadn't noticed the release date on PyPI.

Reply:
haha, interesting, hadn't seen that part of the doc. From what I understand it's talking about using `Bunch` as an input, instead of output. As an output, it's used in a few places, for example: https://github.com/scikit-learn/scikit-learn/blob/76de8833195e8f4ebeaefd65e29e9b429fa2a4e7/sklearn/inspection/_permutation_importance.py#L123

I'll open an issue to discuss the docs about `Bunch` with the other developers.

Reply:
I've updated the PR to use the `Bunch` in `sklearn.utils`. However, I would be happier if there were documentation about it.